coala_quickstart/info_extraction/Information.py

```diff
@@ -0,0 +1,40 @@
+from coala_quickstart.info_extraction.Info import Info
+
+
+class LicenseUsedInfo(Info):
+    description = "License of the project."
+    value_type = (str,)
+
+
+class VersionInfo(Info):
+    description = "Version range information."
```

Comment:
It'd be really neat to have an example value as a docstring for each type of `Info`